# HG changeset patch
# User Rik <rik@octave.org>
# Date 1463462479 25200
#      Mon May 16 22:21:19 2016 -0700
# Node ID 601ee461f23198c92cf03951c71b075b5a99de22
# Parent  efdf6424003af1b55938f9d5e8c756a254e3d583
shading.m: Correct typo in %!demos.

shading.m: Correct "revesed" to "reversed" in %!demos.

diff --git a/scripts/plot/appearance/shading.m b/scripts/plot/appearance/shading.m
--- a/scripts/plot/appearance/shading.m
+++ b/scripts/plot/appearance/shading.m
@@ -109,17 +109,17 @@ endfunction
 %! clf;
 %! colormap ('default');
 %! sombrero ();
 %! shading faceted;
 %! h = findobj (gca (), 'type', 'surface');
 %! facecolor = get (h, 'facecolor');
 %! edgecolor = get (h, 'edgecolor');
 %! set (h, 'edgecolor', facecolor, 'facecolor', edgecolor)
-%! title ({'shading ''faceted''', 'with "edgecolor" and "facecolor" revesed'});
+%! title ({'shading ''faceted''', 'with "edgecolor" and "facecolor" reversed'});
 
 %!demo
 %! clf;
 %! colormap ('default');
 %! sombrero ();
 %! shading flat;
 %! title ('shading ''flat''');
 
@@ -127,17 +127,17 @@ endfunction
 %! clf;
 %! colormap ('default');
 %! sombrero ();
 %! shading flat;
 %! h = findobj (gca (), 'type', 'surface');
 %! facecolor = get (h, 'facecolor');
 %! edgecolor = get (h, 'edgecolor');
 %! set (h, 'edgecolor', facecolor, 'facecolor', edgecolor)
-%! title ({'shading ''flat''', 'with "edgecolor" and "facecolor" revesed'});
+%! title ({'shading ''flat''', 'with "edgecolor" and "facecolor" reversed'});
 
 %!demo
 %! clf;
 %! colormap ('default');
 %! sombrero ();
 %! shading interp;
 %! title ('shading ''interp''');
 
@@ -145,17 +145,17 @@ endfunction
 %! clf;
 %! colormap ('default');
 %! sombrero ();
 %! shading interp;
 %! h = findobj (gca (), 'type', 'surface');
 %! facecolor = get (h, 'facecolor');
 %! edgecolor = get (h, 'edgecolor');
 %! set (h, 'edgecolor', facecolor, 'facecolor', edgecolor)
-%! title ({'shading ''interp''', 'with "edgecolor" and "facecolor" revesed'});
+%! title ({'shading ''interp''', 'with "edgecolor" and "facecolor" reversed'});
 
 %!demo
 %! clf
 %! colormap ('default')
 %! peaks ()
 %! shading interp
 %! h = findobj (gca (), 'type', 'surface');
 %! set (h, 'edgecolor', 'k')
@@ -172,17 +172,17 @@ endfunction
 %! clf;
 %! colormap ('default');
 %! pcolor (peaks ());
 %! shading faceted;
 %! h = findobj (gca (), 'type', 'surface');
 %! facecolor = get (h, 'facecolor');
 %! edgecolor = get (h, 'edgecolor');
 %! set (h, 'edgecolor', facecolor, 'facecolor', edgecolor)
-%! title ({'shading ''faceted''', 'with "edgecolor" and "facecolor" revesed'});
+%! title ({'shading ''faceted''', 'with "edgecolor" and "facecolor" reversed'});
 
 %!demo
 %! clf;
 %! colormap ('default');
 %! pcolor (peaks ());
 %! shading flat;
 %! title ('shading ''flat''');
 
@@ -190,17 +190,17 @@ endfunction
 %! clf;
 %! colormap ('default');
 %! pcolor (peaks ());
 %! shading flat;
 %! h = findobj (gca (), 'type', 'surface');
 %! facecolor = get (h, 'facecolor');
 %! edgecolor = get (h, 'edgecolor');
 %! set (h, 'edgecolor', facecolor, 'facecolor', edgecolor)
-%! title ({'shading ''flat''', 'with "edgecolor" and "facecolor" revesed'});
+%! title ({'shading ''flat''', 'with "edgecolor" and "facecolor" reversed'});
 
 %!demo
 %! clf;
 %! colormap ('default');
 %! pcolor (peaks ());
 %! shading interp;
 %! title ('shading ''interp''');
 
@@ -208,10 +208,10 @@ endfunction
 %! clf;
 %! colormap ('default');
 %! pcolor (peaks ());
 %! shading interp;
 %! h = findobj (gca (), 'type', 'surface');
 %! facecolor = get (h, 'facecolor');
 %! edgecolor = get (h, 'edgecolor');
 %! set (h, 'edgecolor', facecolor, 'facecolor', edgecolor)
-%! title ({'shading ''interp''', 'with "edgecolor" and "facecolor" revesed'});
+%! title ({'shading ''interp''', 'with "edgecolor" and "facecolor" reversed'});
 
